### ISSUE TITLE
Sample data as JSON importable into MongoDB

### ISSUE THREAD

**Issue by dandv** [tidepool-org/hub#117]
Just throwing an idea out there: after starting up services, it would be nice if a sample data dump were available for importing into Mongo. Maybe someone who has publicly-consumable CareLink data could export some of it?

**Comment by jh-bate** [comment]
@dandv good idea -  we do have some data available to open source devs and as you mention it would be good to include that in some sort of a packaged install on a devs machine so they can be up and running and have data to access.
